core/src/index.ts

```diff
@@ -120,6 +124,14 @@ export class Seeder {
         await database.drop();
       }
 
+      if (config.removeAllDocuments) {
+        this.log('Removing all documents from collections...');
+        const promisses = collections.map((collection) =>
+          database!.removeAllDocumentsIfCollectionExists(collection.name),
+        );
+        await Promise.all(promisses);
+      }
+
```

Comment:
- Please move this block before `if (config.dropDatabase)`
- Modify the condition to run it only if the database won't be dropped
- Rename `promisses` to `promises`

Full code block would look like this:
```javascript
     database = await databaseConnector.connect(config.database);

      if (!config.dropDatabase && config.dropCollections) {
        this.log('Dropping collections...');
        for (const collection of collections) {
          await database.dropCollectionIfExists(collection.name);
        }
      }

      if (!config.dropDatabase && config.removeAllDocuments) {
        this.log('Removing all documents from collections...');
        const promisses = collections.map((collection) =>
          database!.removeAllDocumentsIfCollectionExists(collection.name),
        );
        await Promise.all(promisses);
      }

      if (config.dropDatabase) {
        this.log('Dropping database...');
        await database.drop();
      }

      await new CollectionImporter(
        ...
```